ospfd/ospf_apiserver.c

```diff
@@ -80,9 +80,10 @@ struct ospf_interface *ospf_apiserver_if_lookup_by_addr(struct in_addr address)
 {
 	struct listnode *node, *nnode;
 	struct ospf_interface *oi;
-	struct ospf *ospf;
+	struct ospf *ospf = NULL;
 
-	if (!(ospf = ospf_lookup()))
+	ospf = ospf_lookup_by_vrf_id(VRF_DEFAULT);
```

Comment:
ospf-api VRF support will be considered in later phase. For now, type 1-5 and opaque LSAs have been made VRF aware

Reply:
Concerning the ospf_api, I think that the API should report all LSA from all VRF just putting the VRF ID information in top of the LSA.
So that, the function that return the LSA must loop into all VRF ID.

For the other function that allow to push new Opaque LSA, I'm not sure that it will be pertinent to specify a VRF ID. As a first step, we could just mention that the API is only usable for the default VRF.